test/ussd_clinic.test.js

```diff
@@ -507,15 +478,17 @@ describe("app", function() {
         });
 
         describe("dialback sms testing", function() {
+            var setupInputs = [
+                {session_event: 'new'}  // dial in
+                , "1"  // state_start - yes
+                , "1"  // state_consent - yes
+                , {session_event: 'close'}
+            ];
+
             it("send if redial sms not yet sent (identity loads without redial_sms_sent defined)", function() {
                 return tester
                 .setup.user.addr("27820001001")
-                .inputs(
-                    {session_event: 'new'}  // dial in
-                    , "1"  // state_start - yes
-                    , "1"  // state_consent - yes
-                    , {session_event: 'close'}
-                )
+                .inputs.apply(this, JSON.parse(JSON.stringify(setupInputs)))
```

Comment:
I'm confused, why do you suddenly need the stringify and parse calls when the previous tests didn't need them?